distributed/protocol/serialize.py

```diff
@@ -195,6 +202,14 @@ def deserialize(self):
         frames = decompress(self.header, self.frames)
         return deserialize(self.header, frames)
 
+    def __eq__(self, other):
+        return (isinstance(other, Serialized) and
+                other.header == self.header and
+                other.frames == self.frames)
+
+    def __ne__(self, other):
+        return not (self == other)
```

Comment:
We don't in this codebase, that's true, but including objects in containers is generally useful.  I'm fairly confident that I've put serialized objects in containers before while testing.  I'm somewhat confident that this will come up for someone in the future when building something atypical.  If it is cheap to keep this door open then I would prefer to do so.